eos/b-decays/observables.cc

```diff
@@ -2110,6 +2112,126 @@ namespace eos
                         Unit::None(),
                         &BToPiPiLeptonNeutrino::integrated_forward_backward_asymmetry,
                         std::make_tuple("q2_min", "q2_max", "k2_min", "k2_max")),
+
+                make_observable("B^+->pi^+pi^-lnu::BR2", R"(\mathcal{B}(B^-\to \pi^+\pi^- \ell^-\bar\nu))",
```

Comment:
I don't like the naming. This should be ``...::BR(q2_min,q2_max,k2_min,k2_max)``.